### PR TITLE
Remove batch executors from vacuum/dump processes for 6.15 #8051

### PR DESCRIPTION
**Pull request by vbradnitski**

### CODE REVIEW

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/vacuum/versiontable/VersionTableCleanupTask.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5456405&bid=17885672)

File: modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/vacuum/versiontable/VersionTableCleanupTask.java [hunk 1]
```diff
@@ -164,15 +181,27 @@ private boolean versionUsedInAnyBranch( final Repository repository, final NodeV
...
+    private NodeVersionQuery createQuery( final NodeVersionId lastVersionId )
     {
-        final Instant since = Instant.now().minusMillis( params.getAgeThreshold() );
+        final NodeVersionQuery.Builder builder = NodeVersionQuery.create();
+
+        if ( lastVersionId != null )
+        {
+            final RangeFilter versionIdFilter = RangeFilter.create().
+                fieldName( VersionIndexPath.VERSION_ID.getPath() ).
+                from( ValueFactory.newString( lastVersionId.toString() ) ).
```
Comment (rymsha) [inline modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/vacuum/versiontable/VersionTableCleanupTask.java:192]: "from" is inclusive and may end up in infinite loop. In 7.2 Range filter is extended to support includeLower.

Comment (vbradnitski) [reply modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/vacuum/versiontable/VersionTableCleanupTask.java:192]: comment added.

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/vacuum/versiontable/VersionTableCleanupTask.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5456405&bid=17885672)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/vacuum/versiontable/VersionTableCleanupTask.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5456405&bid=17885672)

**Pull request merged**